modules/sqlite_helpers.py

```diff
@@ -126,9 +129,9 @@ def maybe_delete_expired_url(sqlite_file, sqlite_row) -> bool: #returns True if
     utc_tz = ZoneInfo('UTC')
 
     expiration_datetime = None
-    # sqlite_row[5] represents the expiration datetime e.g., "2024-11-04 18:05:24.006593"
+    # sqlite_row[5] represents the expiration datetime in UTC timezone e.g., "2024-11-04 18:05:24"
```

Comment:
maybe make this comment multiline, explaining that all these formats can be parsed now
```
2024-11-04
2024-11-04T18:05:24
2024-11-04T18:05:24.123456
2024-11-04T18:05:24+02:00
```